packages/ng/src/app/select/select.component.ts

```diff
@@ -37,6 +37,8 @@ import {takeUntil} from 'rxjs/operators/takeUntil';
 import {LuSelectOption} from './option/select-option.component';
 import { ISelectClearer } from './clearer/select-clearer.model';
 import { LuSelectClearerComponent } from './clearer/select-clearer.component';
+import { ISelectOptionFeeder } from './option/select-option-feeder.model';
+import { AbstractSelectOptionFeederComponent } from './option/select-option-feeder.component';
```

Comment:
the naming i use is 
 - `IXxx` means interface
 - `AXxx` means abstract class
 - `Xxx` means standard class

so my code usually looks like this

```ts
interface IFoo { ... }
abstract class AFoo implements IFoo { ... }
class Bar extends AFoo implements IFoo { ... }
```